src/DssSpell.sol

```diff
@@ -23,51 +23,15 @@ import "dss-exec-lib/DssAction.sol";
 
-interface DssVestLike {
-    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
-    function restrict(uint256) external;
-}
-
 contract DssSpellAction is DssAction, DssSpellCollateralAction {
 
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
     // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/3f4ec88e5bd6220123e931110784d3b6fee63829/governance/votes/Executive%20vote%20-%20July%2006,%202022.md -q -O - 2>/dev/null)"
     string public constant override description =
-        "2022-07-06 MakerDAO Executive Spell | Hash: 0xbfd59d586d0af2a9d9ea835d09e6b8d2c01b30f5c2ac607592662075f23e72bb";
-
-    DssVestLike immutable MCD_VEST_DAI = DssVestLike(DssExecLib.getChangelogAddress("MCD_VEST_DAI"));
-
-    address constant GRO_001_WALLET = 0x7800C137A645c07132886539217ce192b9F0528e;
-    address constant SF_001_WALLET  = 0xf737C76D2B358619f7ef696cf3F94548fEcec379;
-    address constant KEEP3R_MANAGER = 0xc6A048550C9553F8Ac20fbdeB06f114c27ECcabb;
-
-    address constant FEEDBLACK_LOOPS_WALLET     = 0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1;
-    address constant FLIP_FLOP_FLAP_WALLET      = 0x688d508f3a6B0a377e266405A1583B3316f9A2B3;
-    address constant ULTRASCHUPPI_WALLET        = 0xCCffDBc38B1463847509dCD95e0D9AAf54D1c167;
-    address constant MAKERMAN_WALLET            = 0x9AC6A6B24bCd789Fa59A175c0514f33255e1e6D0;
-    address constant ACRE_INVEST_WALLET         = 0x5b9C98e8A3D9Db6cd4B4B4C1F92D0A551D06F00D;
-    address constant JUSTIN_CASE_WALLET         = 0xE070c2dCfcf6C6409202A8a210f71D51dbAe9473;
-    address constant GFX_LABS_WALLET            = 0xa6e8772af29b29B9202a073f8E36f447689BEef6;
-    address constant DOO_WALLET                 = 0x3B91eBDfBC4B78d778f62632a4004804AC5d2DB0;
-    address constant PENNBLOCKCHAIN_WALLET      = 0x070341aA5Ed571f0FB2c4a5641409B1A46b4961b;
-    address constant FLIPSIDE_CRYPTO_WALLET     = 0x62a43123FE71f9764f26554b3F5017627996816a;
-    address constant CHRIS_BLEC_WALLET          = 0xa3f0AbB4Ba74512b5a736C5759446e9B50FDA170;
-    address constant BLOCKCHAIN_COLUMBIA_WALLET = 0xdC1F98682F4F8a5c6d54F345F448437b83f5E432;
-    address constant MHONKASALO_TEEMULAU_WALLET = 0x97Fb39171ACd7C82c439b6158EA2F71D26ba383d;
-    address constant GOVERNANCE_HOUSE_WALLET    = 0x82cD339Fa7d6f22242B31d5f7ea37c1B721dB9C3;
-
-    // Friday, 1 July 2022 00:00:00
-    uint256 constant JUL_01_2022 = 1656633600;
-    // Tuesday, 31 January 2023 00:00:00
-    uint256 constant JAN_31_2023 = 1675123200;
-    // Friday, 30 June 2023 00:00:00
-    uint256 constant JUN_30_2023 = 1688083200;
-    // Saturday, 1 July 2023 00:00:00
-    uint256 constant JUL_01_2023 = 1688169600;
-
-    // Math
-    uint256 constant WAD = 10**18;
+        "2022-07-13 MakerDAO Executive Spell | Hash: TODO";
+
+    uint256 constant MILLION = 10 ** 6;
```

Comment:
Consider specifying visibility (e.g. `internal`)

Reply:
Think I'm okay just leaving it out.